modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -258,6 +259,136 @@ class window.ACASFormLSCodeValueFieldController extends ACASFormAbstractFieldCon
 
 		@
 
+class window.ACASFormLSThingInteractionFieldController extends ACASFormAbstractFieldController
+	###
+		Launching controller must:
+		- Initialize the model with an LSInteraction
+    Do whatever else is required or optional in ACASFormAbstractFieldController
+	###
+	events: ->
+		"change select": "handleInputChanged"
+
+	template: _.template($("#ACASFormLSThingInteractionFieldView").html())
+	@
```

Comment:
Seems like there shouldn't be an "@" sign hanging out here, or is this just weird github formatting artifact?